src/lib/base/Log.cpp

```diff
@@ -159,20 +161,21 @@ const char *Log::getFilterName() const
   return getFilterName(getFilter());
 }
 
-const char *Log::getFilterName(int level) const
+const char *Log::getFilterName(LogLevel level) const
 {
-  if (level < 0) {
+  const auto lvl = static_cast<int>(level);
```

Comment:
Perhaps this would be clearer for intent.
```suggestion
  const auto levelIndex = static_cast<int>(level);
```